packages/csv/README.md

```diff
@@ -0,0 +1,29 @@
+# CSV
+
+WooCommerce utility library to convert data to CSV files.
+
+## Installation
+
+Install the module
+
+```bash
+npm install @woocommerce/csv --save
+```
+
+## Usage
+
+```js
+onClick = () => {
+	// Triggers a browser UI to save a file, named the first argument, with the contents of
+	// the second argument.
+	downloadCSVFile(
+		// Create a file name based on the report title and optional query. Will return a
```

Comment:
`downloadCSVFile()` can be used for any kind of CSV data, because it just requires the file name and the CSV contents already formatted. Same for `generateCSVFileName()`.

`generateCSVDataFromTable()` requires a `headers` and `rows` params (headers must have a `label` attribute and rows must have a `value` attribute), but as long as this is respected, any tabular data --not only reports-- could use this function.